.eslintrc

```diff
@@ -4,12 +4,13 @@
     "jasmine": true,
     "node": true
   },
+  "extends": ["plugin:prettier/recommended"],
   "rules": {
     "camelcase": [2, {"properties": "always"}],
     "indent": [2, 2, {"SwitchCase": 1}], // 2 spaces indentation
     "semi": [2, "always"],
     "keyword-spacing": [2],
-    "space-before-function-paren": [2, "never"],
+    "space-before-function-paren": [2, "always"],
```

Comment:
If you remember, I told you that prettier 2.0 adds now a space before function parenthesis. Because of that, eslint reports errors. I think I configured prettier correctly. Alternatives would be to stick with prettier 1.0 (it does not add a space before function parenthesis). Or we run an eslint fix command before any commit.

Reply:
I think we need the `prettier` eslint plugin (https://github.com/prettier/eslint-plugin-prettier) so that eslint doesn't try to cover formatting things. That's what we did with Alloy anyway.